package.json

```diff
@@ -34,8 +34,9 @@
         "jju": "^1.3.0",
         "jquery": "^3.3.1",
         "json-stringify-safe": "^5.0.1",
-        "lodash": "^4.17.10",
-        "mathjax": "^2.7.4",
+        "lodash": "^4.17.4",
+        "lru-cache": "^4.1.3",
+        "mathjax": "^2.7.2",
```

Comment:
downgraded lodash and mathjax?

Reply:
Don't know why that happened - rebase gone wrong, perhaps? Will fix.